pkg/util/BUILD.bazel

```diff
@@ -4,9 +4,13 @@ go_library(
     name = "go_default_library",
     srcs = [
         "hw_helper.go",
+        "os_utils.go",
```

Comment:
As this is under the `util` folder, re-using the name internally does not add any valuable info.
Maybe just `os`? It is not a package name, so there should not be any namespace collision here.
Other options: `os_extender`, `os_extra`.